stingray/modeling/parameterestimation.py

```diff
@@ -297,34 +290,73 @@ def fit(self, lpost, t0, neg=True, scipy_optimize_options=None):
             params = [getattr(newmod,name) for name in newmod.param_names]
             bounds = [p.bounds for p in params if not np.any([p.tied, p.fixed])]
 
+            if len(bounds) > 0 and self.fitmethod not in ["L-BFGS-B",
+                                                          "TNC",
+                                                          "SLSQP"]:
+                logging.warning("Fitting method %s "%self.fitmethod +
+                                "cannot incorporate the bounds you set!")
+
+            if len(bounds) == 0 or self.fitmethod not in ["L-BFGS-B",
```

Comment:
But why are we setting bounds in tests that shouldn't have them? Or else, why are the warnings thrown if we are using the correct bounded methods? 
Otherwise sure, I see your point